packages/expo-router/src/views/Try.tsx

```diff
@@ -23,9 +22,6 @@ export class Try extends Component<
   state = { error: undefined };
 
   static getDerivedStateFromError(error: Error) {
-    // Force hide the splash screen if an error occurs.
-    SplashScreen.hideAsync();
```

Comment:
Splash screen will stay open and cover error messages.

Reply:
These cannot be detected outside of the component tree, it stops bubbling when the user catches the error to show their own error boundary.